SimpCityTests/GameTests.cs

```diff
@@ -2,18 +2,21 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SimpCity;
 
-namespace SimpCityTests {
+namespace SimpCityTests
+{
```

Comment:
Please run the formatter again after merging the `main` branch as cecce1c211e5fec5e19935c3a0d31b5d80928e6f updates the format spec.